docs/provisioner/01-01-provisioner-overview.md

```diff
@@ -16,7 +16,7 @@ It is powered by [Hydroform](https://github.com/kyma-incubator/hydroform) and it
     
 Note that the operations of provisioning and deprovisioning are asynchronous. They return the operation ID, which you can use to [check the Runtime Operation Status](08-03-runtime-operation-status.md).
 
-The Runtime Provisioner also allows you to [clean up Runtime data](08-06-clean-up-runtime-data.md). This operation removes a given Runtime and all its data from the database and frees up the Runtime ID for reuse. It is useful when your cluster has died or when the operation of deprovisioning has failed.
+The Runtime Provisioner also allows you to [clean up Runtime data](08-06-clean-up-runtime-data.md). This operation removes the data for a given Runtime from the database and frees up the Runtime ID for reuse. It is useful when your cluster has died or when the operation of deprovisioning has failed.
```

Comment:
I would suggest adding some `Caution` section informing the user that this operation does not trigger Runtime deprovisioning and cluster might still exist after running it.